README.md

```diff
@@ -8,7 +8,17 @@
 
 Blobstream is a Celestia -> EVM message relay.
 It is based on Umee's Gravity Bridge implementation, [Peggo](https://github.com/umee-network/peggo).
-**This project is under active development and should not be used in production**.
+
+## ⚠️ DEPRECATED
+
+**This Blobstream contract implementation is deprecated and not maintained. It may contain bugs and should not be used in production.**
```

Comment:
replace: This Blobstream contract implementation is deprecated by stating exactly: The [Blobstream contract](link) is deprecated ...

Also, change the blobstream.sol contract to have the deprecated tag in solidity